tests/irepela/homework_4/test_list_methods.py

```diff
@@ -24,5 +24,7 @@ def test_list_index(a, b, expected):
 
 
 def test_list_extend():
-    some_list.extend([7, 8, 9])
-    assert some_list == [1, 2, 3, 4, 5, 6, 1, 3, 2, 5, 4, 7, 8, 9]
+    list_copy = some_list.copy()

```

Comment:
I prefer:

list(some_list)

since it is more flexible way to copy elements from set, tuple, list etc,  to list